xml/install_caasp_heat_templates.xml

```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
    When you have successfully accessed the admin node web interface via the
    floating IP, follow the instructions at <link
    xlink:href="https://www.suse.com/documentation/suse-caasp-3/book_caasp_deployment/data/book_caasp_deployment.html"/> to
-   continue the Setup of &caasp;.
+   continue the setup of &caasp;.
   </para>
  </section>
+ <section>
+  <title>Installing &caasp; with Multiple Masters</title>
+  <note>
+   <para>
+    A &o_orch; stack with load balancing and multiple master nodes can only be
+    created from the command line, because &o_dash; does not have support for nested
+    &o_orch; templates.
+   </para>
+  </note>
+  <para>
+   Install the <package>caasp-openstack-heat-templates</package> package on a
+   machine with &productname; repositories:
+  </para>
+  <screen>zypper in caasp-openstack-heat-templates</screen>
+  <para>
+   The installed templates are located in
+   <filename>/usr/share/caasp-openstack-heat-templates</filename>.
+  </para>
+  <para>
+   A working load balancer is needed in your &cloud; deployment. Two
+   load balancer providers are available: Octavia and HAProxy.
+  </para>
+  <para>
+   Verify that load balancing with Octavia or HAProxy is working correctly
+   in your &ostack; installation by creating a load balancer manually and
```

Comment:
install by* reads better to me, thoughts?

Reply:
installation works for me

Reply:
Agreed a few hours later